### PR TITLE
Add documentation about how to use IAM roles

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY
Document the two ways of using IAM roles in plugins.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
lib/ansible/plugins/inventory/aws_ec2.py

### CODE REVIEW

File: lib/ansible/plugins/inventory/aws_ec2.py [hunk 1]
```diff
@@ -17,13 +17,18 @@
...
+        - If no credentials are provided and the control node has an associated IAM instance profile then the
+          role will be used for authentication.
     options:
         plugin:
             description: token that ensures this is a source file for the 'aws_ec2' plugin.
             required: True
             choices: ['aws_ec2']
         boto_profile:
-          description: The boto profile to use.
+          description: The boto profile to use. This plugin supports boto3-style credentials, so the profile
```
Comment (kbreit) [inline lib/ansible/plugins/inventory/aws_ec2.py]: I normally split this sort of statement onto multiple lines. Something similar to...

```
description:
- The boto profile to use.
- This plugin supports boto3-style...
- See U(...
```

Comment (s-hertel) [reply lib/ansible/plugins/inventory/aws_ec2.py]: That's tidier, good idea. I updated the plugin so it would be consistent.

**Comment by kbreit** [comment]
lgtm

**Review by kdelee — APPROVED** [review]
LGTM, thanks!

**Pull request merged**